src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -141,7 +170,18 @@ internal abstract partial class DefinitionItem
         {
             return Create(
                 tags, displayParts, sourceSpans, nameDisplayParts,
-                properties: null, displayIfNoReferences: displayIfNoReferences);
+                properties: null, displayableProperties: ImmutableArray<FindUsageProperty>.Empty, displayIfNoReferences: displayIfNoReferences);
+        }
+
+        public static DefinitionItem Create(
+            ImmutableArray<string> tags,
+            ImmutableArray<TaggedText> displayParts,
+            ImmutableArray<DocumentSpan> sourceSpans,
+            ImmutableArray<TaggedText> nameDisplayParts = default,
+            ImmutableDictionary<string, string> properties = null,
```

Comment:
name is inconsistent.  this should either be `displayableProperties` or we should rename the field to `Properties`.

Reply:
This is the original `Properties` property as explained https://github.com/dotnet/roslyn/pull/37802#discussion_r324846036.

Reply:
gotcha.  thanks!